db/Untitled.sql

```diff
@@ -0,0 +1,17 @@
+drop database if exists teste;
```

Comment:
Remove this file from the PR, please

Reply:
I removed it, I don't even know why is was there